tests/catch/hipTestMain/config/config_amd_linux.json

```diff
@@ -1,7 +1,12 @@
 {
-    "DisabledTests":
-	[
-	   "Unit_hipStreamPerThread_DeviceReset_1"
-	]
-
+  "DisabledTests" : [
+    "Unit_hipStreamPerThread_DeviceReset_1", "# Following test fails due to EXSWCPHIPT-59",
+    "Unit_hipMallocArray_Negative_ChannelAfterZeroChannel",
+    "# Following test fails due to EXSWCPHIPT-59",
```

Comment:
Shouldn't those be comments?

Reply:
Unfortunately there are no comments available in JSON, so we thought this would be a good compromise. From experience it doesn't seem to affect catch2 or ctest.

Reply:
Can't we just put that info inside the test source code then?

Reply:
Sure, I've sorted that out.